docs/README.md

```diff
@@ -1,13 +1,13 @@
 # How to contribute documentation
 
-Thank you very much for contributing to StarRocks documentation! Your help is important to our improvement!
+Thank you very much for contributing to StarRocks documentation! Your help is important to help improve the docs!
 
 Before contributing, please read this article carefully to quickly understand the tips, writing process, and documentation templates.
 
 ## Tips
 
 1. Language: Please use at least one language, Chinese or English. The bilingual version is highly preferred.
-2. Index: When you add a topic, you also need to add an index for the topic in the TOC file, for example, `[introduction](/introduction/StarRocks_intro.md)`. **The topic path must be an absolute path.** This TOC file will eventually be rendered as the side navigation bar for documentation on our official website.
+2. Index: When you add a topic, you also need to add an entry for the topic in the table of contents(TOC) file, for example, `[introduction](/introduction/StarRocks_intro.md)`. **The topic path must be an absolute path.** This TOC file will eventually be rendered as the side navigation bar for documentation on our official website.
```

Comment:
the original message may be wrong. In English docs, we use relative paths, pls help correct.